src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -0,0 +1,1232 @@
+
+        final VariantContextBuilder builder = new VariantContextBuilder(mergedVC);
+
+        builder.attribute(GATKVCFConstants.AS_VQS_LOD_KEY, relevantVqsLodMap.values().stream().map(val -> val.equals(Double.NaN) ? VCFConstants.EMPTY_INFO_FIELD : val.toString()).collect(Collectors.toList()));
+        builder.attribute(GATKVCFConstants.AS_YNG_STATUS_KEY, new ArrayList<>(relevantYngMap.values()));
+
+        if (VQSLODFilteringType.equals(ExtractCohort.VQSLODFilteringType.SITES)) { // Note that these filters are not used with Genotype VQSLOD Filtering
+            int refLength = mergedVC.getReference().length();
+
+            // if there are any Yays, the site is PASS
+            if (remappedYngMap.containsValue("Y")) {
+                builder.passFilters();
+            } else if (remappedYngMap.containsValue("N")) {
+                builder.filter(GATKVCFConstants.NAY_FROM_YNG);
+            } else {
+                // if it doesn't trigger any of the filters below, we assume it passes.
+                builder.passFilters();
+                if (remappedYngMap.containsValue("G")) {
+                    Optional<Double> snpMax = relevantVqsLodMap.entrySet().stream()
+                            .filter(entry -> entry.getKey().length() == refLength)
+                            .map(entry -> entry.getValue())
+                            .filter(d -> !(d.isNaN()||d.isInfinite()))
+                            .max(Double::compareTo);
+                    if (snpMax.isPresent() && snpMax.get() < vqsLodSNPThreshold) {
+                        builder.filter(GATKVCFConstants.VQSR_FAILURE_SNP);
+                    }
+
+                    Optional<Double> indelMax = relevantVqsLodMap.entrySet().stream()
+                            .filter(entry -> entry.getKey().length() != refLength)
+                            .map(entry -> entry.getValue())
+                            .filter(d -> !(d.isNaN()||d.isInfinite()))
+                            .max(Double::compareTo);
+
+                    if (indelMax.isPresent() && indelMax.get() < vqsLodINDELThreshold) {
+                        builder.filter(GATKVCFConstants.VQSR_FAILURE_INDEL);
+                    }
+                } else {
+                    // per-conversation with Laura, if there is no information we let the site pass (ie no data does not imply failure)
+                }
+            }
+        }
+
+        // TODO: add in other annotations we need in output (like AF, etc?)
+        final VariantContext filteredVC = builder.make();
+        return filteredVC;
+    }
+
+    protected static VariantContext removeAnnotations(VariantContext vc) {
+
+        final VariantContextBuilder builder = new VariantContextBuilder(vc);
+        List<String> rmAnnotationList = new ArrayList<>(Arrays.asList(
+                GATKVCFConstants.STRAND_ODDS_RATIO_KEY,
+                GATKVCFConstants.AS_QUAL_BY_DEPTH_KEY,
+                GATKVCFConstants.FISHER_STRAND_KEY));
```

Comment:
add to this list:
                GATKVCFConstants.EXCESS_HET_KEY,


(and also add to the unit test testRemoveAnnotations in  ExtractCohortRemoveAnnotationsUnitTest:
        Assert.assertFalse(expectedVC.hasAttribute(GATKVCFConstants.EXCESS_HET_KEY), "removeAnnotations did not remove ExcessHet annotation.");
)